ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/transaction/VertxNonceRequestTransmitter.java

```diff
@@ -80,7 +84,7 @@ public BigInteger requestNonce(final JsonRpcRequest request) {
     final HttpClientRequest request =
         client.request(
             HttpMethod.POST,
-            "/",
+            downstreamPathCalculator.calculateDownstreamPath(""),
```

Comment:
Why empty string instead of / here? Don't we want the path to be / in this case?